spec/services/location_maps_generator_service_spec.rb

```diff
@@ -46,5 +46,16 @@
       expect(File.exist?(locations_path)).to be true
       expect(File.exist?(location_display_path)).to be true
     end
+
+    context 'with existing template files' do
+      before do
+        allow(FileUtils).to receive(:cp)
+      end
+
+      it 'overwrites the templates' do
+        described_class.generate_from_templates
+        expect(FileUtils).to have_received(:cp).exactly(3).times
+      end
+    end
```

Comment:
we want to overwrite the 2 templates. The one is custom and solid and we dont create the 3rd .rb for this one.

Reply:
I'm talking about this template. Should not be overwritten. https://github.com/pulibrary/bibdata/blob/1060-templates/marc_to_solr/translation_maps/holding_library.rb.tmpl

Reply:
@christinach The templates are not overridden. Only the `.rb` translation maps. I restructured the test so it works better and is clearer.

Reply:
If you think that we should check in `holding_library.rb` to the repo instead of copying from a template, I would open a new issue.

Reply:
> @christinach The templates are not overridden. Only the `.rb` translation maps. I restructured the test so it works better and is clearer.

Thanks @eliotjordan !

Reply:
> If you think that we should check in `holding_library.rb` to the repo instead of copying from a template, I would open a new issue.

We should copy from the template.